en/advanced/parameters_and_configurations.md

```diff
@@ -330,6 +333,26 @@ The purpose of each line is given below (for more detail see [module_schema.yaml
  */
 ```
 
+#### Publishing Parameter Metadata to a GCS
+
+Parameter metadata is collected into a JSON or XML file during each PX4 build.
+For all targets with enough FLASH available the JSON file is xz-compressed and stored within the generated binary.
+The file is then shared to ground stations using the [MAVLink Component Information Protocol](https://mavlink.io/en/services/component_information.html).
+This ensures that parameter metadata is always up-to-date with the code running on the vehicle.
+
+Binaries for flight controller targets with constrained memory (`CONSTRAINED_MEMORY`) do not store the *build-specific* parameter metadata in the binary, but instead reference metadata stored on `px4-travis.s3.amazonaws.com`.
```

Comment:
What is `CONSTRAINED_MEMORY`? It doesn't appear to be a parameter. 

Note, this is my interpretation of your original text - ie on constrained target you use the hosted metadata NOT the build specific metadata. If you want to have custom params on resource constrained platform then you have to upload your custom metadata to your own server - below we show how to set your own server.

Reply:
Example: https://github.com/PX4/PX4-Autopilot/blob/master/boards/omnibus/f4sd/default.cmake#L9

It's hosted, but still build-specific (CI uploads all targets).

Reply:
Thanks for clarification. I have slightly reworded.